### ISSUE TITLE
Qemu connection to tap interface fails after starting the device

### ISSUE THREAD

**Issue by cioby23** [GNS3/gns3-gui#554]
I have an vIOS-L3 image (Qemu) connected to a tap0 interface using the Cloud device and whenever I try to start the device after 2 seconds it stops and return the following error message.
=> R1 has started
QEMU process has stopped, return code: 1
Start QEMU with /usr/bin/qemu-system-x86_64 -name R1 -m 384 -enable-kvm -hda /home/cioby/GNS3/Projects/Cisco Discovery Protocol (CDP) Lab/project-files/qemu/b84fa90c-e03b-4afa-8e5a-a6a39b939dee/hda_disk.qcow2 -serial telnet:127.0.0.1:2007,server,nowait -monitor tcp:127.0.0.1:53988,server,nowait -nographic -net none -device e1000,mac=00:00:ab:9d:ee:00,netdev=gns3-0 -netdev tap,id=gns3-0,ifname=tap0 -device e1000,mac=00:00:ab:9d:ee:01 -device e1000,mac=00:00:ab:9d:ee:02,netdev=gns3-2 -netdev socket,id=gns3-2,udp=127.0.0.1:10005,localaddr=127.0.0.1:10004 -device e1000,mac=00:00:ab:9d:ee:03,netdev=gns3-3 -netdev socket,id=gns3-3,udp=127.0.0.1:10019,localaddr=127.0.0.1:10018 -device e1000,mac=00:00:ab:9d:ee:04 -device e1000,mac=00:00:ab:9d:ee:05 -device e1000,mac=00:00:ab:9d:ee:06 -device e1000,mac=00:00:ab:9d:ee:07

Execution log:
/etc/qemu-ifup: could not launch network script
qemu-system-x86_64: -netdev tap,id=gns3-0,ifname=tap0: Device 'tap' could not be initialized

Basically after starting the Qemu VM the tap interface loses it's IP address.
I'm using the latest development version 1.4.0 dev2 on Linux with Qemu 2.3.0. If you need more details please let me know.
![qemu_tap](https://cloud.githubusercontent.com/assets/7124193/8763192/0e9d0bb8-2d9a-11e5-9f9f-d6e2206387e9.png)

**Comment by noplay** [comment]
Hi,

Thanks for the report.

With 1.3.7 or the 1.4alpha1?

**Comment by cioby23** [comment]
I'm using the latest development version 1.4.0 dev2

**Comment by noplay** [comment]
Thanks

**Comment by grossmj** [comment]
Look like this is a problem on your system which may be related to https://bugs.launchpad.net/ubuntu/+source/qemu-kvm/+bug/475327

I've tested Qemu VM to TAP and found no issue on my Linux.

**Comment by cioby23** [comment]
I managed to fix this issue. I'm using Archlinux 64 bit.
The issue was that the /etc/qemu-ifup and /etc/qemu-ifdown scripts were missing so I have created them manually using https://wiki.archlinux.org/index.php/QEMU#Creating_bridge_manually.
By default Qemu on Archlinux uses the qemu-bridge-helper binary which allows creating tap devices on an existing bridge, but I didn't have one created before. Details on https://wiki.archlinux.org/index.php/QEMU#Bridged_networking_using_qemu-bridge-helper
